django/db/models/expressions.py

```diff
@@ -982,8 +982,7 @@ def get_source_expressions(self):
     def as_sql(self, compiler, connection):
         connection.ops.check_expression_support(self)
         expression_sql, params = compiler.compile(self.expression)
-        placeholders = {'expression': expression_sql}
-        placeholders['ordering'] = 'DESC' if self.descending else 'ASC'
+        placeholders = {'expression': expression_sql, 'ordering': 'DESC' if self.descending else 'ASC'}
```

Comment:
While you're making changes for readability, I would find it best to declare one key-value pair on each line, like you did in the other diff.